benchmark/rclcpp/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
I should we should add a README to tell testers how to use `ament` utility to build and install these cpp programs.

Reply:
Thanks for correcting some stupid errors, I will update the PR later.